modules/npmscripts.config.js

```diff
@@ -26,6 +26,144 @@ const CHECK_AND_FIX_GLOBS = [
 
 module.exports = {
 	check: CHECK_AND_FIX_GLOBS,
+	defaults: {
+		federation: {
+			mode: 'disabled',
+			remotes: [
```

Comment:
I understand this makes it easier for us to make updates in DXP but at the same time it might make it harder to make this public (or maybe not). That was one of the goals of our presets, so external devs could share our build process. This is a step in the opposite direction which I assume we'll need to correct in a different way in the future.

![just-sayin](https://user-images.githubusercontent.com/905006/109158586-7995d280-776b-11eb-869c-a9995541168e.gif)

Reply:
Well, we are going to have the default settings in the root folder of the workspace. That should work as a preset.

Also, because the config files are `.js`, if the users want to have predefined sets of settings, they may as well publish them as packages/modules and simply retrieve them with `require()` in the config files (as we are already doing in some portal files).

That also makes the model much more powerful, because you (as developer) are not limited to one preset, but you can also merge several following the strategy you want. I removed presets in bundler 3 for the very same reason.

I bet @wincent's left arm (just in case :roll_eyes: ) we won't need them.